File: Source/Fantasy.Jobs/JobException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

namespace Fantasy.Jobs
{
    [Serializable]
    public class JobException : Exception
    {

        protected JobException(SerializationInfo info, StreamingContext context)
            : base(info, context)
        {
        }

        public JobException()
        {

        }
        public JobException(string message, Exception innerException)
            : base(message, innerException)
        {

        }

        public JobException(string message)
            : base(message)
        {

        }
    }
}

File: AutoConverted/Fantasy.Jobs/JobException.java
package Fantasy.Jobs;

public class JobException extends RuntimeException implements Serializable
{

	protected JobException(SerializationInfo info, StreamingContext context)
	{
		super(info, context);
	}

	public JobException()
	{

	}
	public JobException(String message, RuntimeException innerException)
	{
		super(message, innerException);

	}

	public JobException(String message)
	{
		super(message);

	}
}